packages/eas-cli/src/graphql/queries/AppleDeviceQuery.ts

```diff
@@ -0,0 +1,91 @@
+import gql from 'graphql-tag';
+
+import { graphqlClient, withErrorHandlingAsync } from '../client';
+import { AppleDevice, AppleTeam } from '../generated';
+
+export type AppleTeamQueryResult = Pick<AppleTeam, 'id' | 'appleTeamIdentifier' | 'appleTeamName'>;
+
+export type AppleDeviceQueryResult = Pick<
+  AppleDevice,
+  'id' | 'identifier' | 'name' | 'deviceClass' | 'enabled'
+>;
+
+export type AppleDevicesByAccountNameQueryResult = AppleTeamQueryResult & {
+  appleDevices: AppleDeviceQueryResult[];
+};
+
+export type AppleDevicesByIdentifierQueryResult = AppleDeviceQueryResult & {
+  appleTeam: AppleTeamQueryResult;
+};
+
+const AppleDeviceQuery = {
```

Comment:
I think it'd be better to have a query like this:
```
{
  account {
    byName(accountName: "dsokal-staging") {
      appleTeams(appleTeamIdentifier: "QL76XYH73P") {
        id
        appleTeamName
        appleDevices {
          id
          identifier
          name
          deviceClass
          enabled
        }
      }
    }
  }
}
```

Reply:
This way you don't need the account id and the query you added in a new file - `packages/eas-cli/src/graphql/queries/AccountQuery.ts`